placement/placement_test.go

```diff
@@ -160,23 +160,23 @@ func TestValidate(t *testing.T) {
 }
 
 func TestSnapshotMarshalling(t *testing.T) {
-	invalidJSON := `[
-		{"ID":123,"Rack":"r1","Shards":[0,7,11]}
-	]`
+	invalidJSON := `{
+		:{"ID":123,"Rack":"r1","Shards":[0,7,11]}
+	}`
 	data := []byte(invalidJSON)
 	ps, err := NewPlacementFromJSON(data)
 	assert.Nil(t, ps)
 	assert.Error(t, err)
 
-	validJSON := `[
```

Comment:
no it's not, but apparently the diff is smaller with maps if we change the placements